### PR TITLE
Create 23-weight-loss.lisp

### PR DESCRIPTION
**Pull request by Puellula**
Unintentional weight loss likely
unintentional weight loss of > 5% of body weight. 
Calculate % weight loss = (previous weight - current weight) ÷ previous weight  x 100 
TB, diabetes, HIV
Nausea or vomiting
Loss of appetite
Stress or anxiety
No money for food
Life limiting illness
Abdominal pain, diarrhoea, constipation 

Review medication: anticonvulsants, antidepressants, diabetes medications and levothyroxine can cause unintentional weight loss. Discuss with doctor.

Refer to social worker 
-Organise nutritional support
-Neglect in the older 
-Ill patient needing care

Thyroid disease likely 
Pulse ≥ 100, palpitations, tremor, dislike of hot weather or thyroid enlargement, 
OR
Tiredness, muscle weakness, weight gain, low mood, dry skin, constipation or cold intolerance

Refer for Doctor to confirm muscle weakness 

Cervical cancer likely
Unintentional weight loss, Abnormal vaginal  discharge/bleeding

Breast cancer likely
Unintentional weight loss, Breast lump/s or nipple discharge

Prostate cancer likely 
Unintentional weight loss, Urinary symptoms in man

 Refer same week:
-Hard, nodular prostate

Bowel cancer likely
Unintentional weight loss, Change in bowel habit

Refer same week
-Mass on abdominal examination 
-Mass on rectal examination 
-Stool occult blood positive

Lung cancer likely
Unintentional weight loss, Cough ≥ 2 weeks, blood-stained sputum, long smoking history

Refer same week
-Suspicious chest x-ray

Oral/esophageal candida likely
Sore mouth, difficulty swallowing 

Depression likely 
in the past month, has patient: 
1)felt down, depressed, hopeless 
2) felt little interest or pleasure in doing things

Alcohol/drug use likely 
In the past year, has patient: 
1) drunk ≥ 4 drinks1/session
2) used illegal drugs 
3) misused prescription or over-the-counter medication

### CODE REVIEW

File: s_expression/system_diagnosis_rules/apc-adult/23-weight-loss.lisp [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+		(clinical_finding (snomed_concept “Hard prostate” (finding))
+		(clinical_finding (snomed_concept “Nodular prostate without urinary obstruction” (disorder))
+		)
+	)
+)
+;;Page 23 – Weight loss
+(system_diagnosis_rule
+	"Bowel cancer likely"
+	(adult
+	(clinical_finding (snomed_concept “Malignant neoplasm of bowel” (disorder))
```
Comment (Puellula) [inline s_expression/system_diagnosis_rules/apc-adult/23-weight-loss.lisp:155]: Replace with 

        (clinical_finding (snomed_concept “Malignant neoplasm of intestine” (disorder))

File: s_expression/system_diagnosis_rules/apc-adult/23-weight-loss.lisp [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+		(clinical_finding (snomed_concept “Unintentional weight loss” (finding))
+		(clinical_finding (snomed_concept “Altered bowel function” (finding))
+		)
+	)
+;;Page 23 – Weight loss
+(system_diagnosis_rule
+	"Bowel cancer likely"
+	(adult
+	(referral 
+	(clinical_finding (snomed_concept “Malignant neoplasm of bowel” (disorder))
```
Comment (Puellula) [inline s_expression/system_diagnosis_rules/apc-adult/23-weight-loss.lisp:168]: Replace with
        (clinical_finding (snomed_concept “Malignant neoplasm of intestine” (disorder))

**Review by Puellula — COMMENTED** [review]
Bowel cancer is a an alias correct terminology is malignant neoplasm of intestine
